src/deploy/table_report_policy.sql

```diff
@@ -0,0 +1,23 @@
+-- Deploy maevsi:table_report_policy to pg
+
+BEGIN;
+
+GRANT INSERT, SELECT ON TABLE maevsi.report TO maevsi_account;
```

Comment:
I can imagine that users without an account should be able to create reports for problematic public content too, don't they?

Reply:
Yes and no...
Reporting yes, but blocking not possible as no linked account. Report action could lead to registration flow and request must be marked as "external" for evaluation (no automated flagging of UGC by externals).
For "internal" request a report should lead to immediate blocking of that UGC for the reporter.

Reply:
Ok, this PR is about reporting as a first step

Reply:
Is there any specific legal reason why a report should lead to a block right away or is this a design decision which makes sense?

Reply:
It's a design decision, a personal conversation concluded.